src/Codeception/Attribute/After.php

```diff
@@ -7,4 +7,7 @@
 #[Attribute(Attribute::TARGET_METHOD | Attribute::IS_REPEATABLE)]
 final class After
 {
+    public function __construct(string $methodName)
```

Comment:
Repeatable attribute should use `...$methodNames` , like in #6600 , right?

Reply:
I'm going to drop my commit in favour of @mdoelker's work in #6600 as that looks better than mine. I'll add a handful of suggestions to that PR for a few tweaks to make it perfect.